cli/cmd/plugin/ui/web/tanzu-ui/src/shared/components/wizard/StepNav.tsx

```diff
@@ -28,9 +27,12 @@ function StepNav(props: ChildProps | any) {
                         key={index}
                         style={{ width: `${100 / totalSteps}%` }}
                         onClick={() => {
-                            if (!hasInvalidStep) {
```

Comment:
nit: to make the code more readable to a new developer, what do you think of

1. creating a method `existsPreviousInvalidStep(index)` and 
2. using lines 30-34  in the implementation
3. using the original logic but instead of `!hasInvalidStep` use `!existsPreviousInvalidStep(index)`